### PR TITLE
hello- icroverse-feature

### PR DESCRIPTION
**Pull request by Berucan**
I added  HTML file  with h1 tag within the header element, css file for styling the h1 element within the header element and also modified the README file  by adding the project description and social network links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+    <header class="headerOne">
+        <h1>Hello Microverse!</h1>
+    </header>
+  <body>
+  </body>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, use correct structure of HTML file. You should also close `html` tag. Also, please, insert all tags inside of `body` tag. Your `header` tag is outside of `body` tag.
- You can see this issue information in linter error
Screen Shot 2022-01-06 at 10 46 02

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider, that a `viewport` meta element is not specified.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.headerOne{
+    color:aqua;
+}
```
Comment (elmar8287) [inline style.css]: - [ ] Please, consider linter errors
Screen Shot 2022-01-06 at 10 50 12

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Berucan ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [x] Please, consider comments under review
- [x] Please, modify your README file, make it more informative. My suggestions are to add your information of social network links, delete some empty, not-needed lines.
Screen Shot 2022-01-06 at 10 45 06

- [ ] Please, consider that the PR (Pull Request) should have a descriptive title telling you what changes are inside.
- [ ] Please, consider that the PR should have a short summary describing introduced changes in more detail. You have not provided a description.


#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello Microverese</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
```
Comment (elmar8287) [inline index.html:7]: - [ ] Here you have again an issue with a linter. It asks you to add a `viewport ` meta tag. Please, add into your `head` section the following:

` <meta name="viewport" content="width=device-width, initial-scale=1.0">` and also correct the link to your .css file. This is correct one `<link rel="stylesheet" href="style.css">`

Please, also add `lang="en"` in HTML tag in beginning. It should looks like this `<html lang="en">` - you can fix by this the warning message from linter

File: style.css [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+.headerOne { color: aqua; }
+
+
```
Comment (elmar8287) [inline style.css:4]: - [ ] Please, fix the linter, as I mentioned in the previous review. You have empty lines in it. Please, before pushing it, you can run `npx stylelint "**/*.{css,scss}"` command in terminal and check any linter errors and fix them before.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Berucan ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider comments under review
- [ ] Please, fix your PR title and description, as I mentioned in the previous review. Do it more informative. Something like `In this PR I added HTML file with h1 tag with ..... ` and also `I modified README file, add the project description ....` - hope you understand my suggestions.

This is your PR:

Screen Shot 2022-01-06 at 15 42 30
Please, edit the title and description. 


#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Berucan** [comment]
Good morning,
I wanted to say thank you for all the review comments you gave me. I got my
work approved.

**Pull request closed without merging**